rotkehlchen/tests/api/test_bitcoin.py

```diff
@@ -57,29 +58,6 @@
 ]
 
 
-def _check_xpub_addition_outcome(outcome, xpub):
```

Comment:
undo everything that has to do with xpubs here. As mentioned above this is the only place this is done/tested

Reply:
Undone